corehq/apps/ota/tests/test_formplayer_restore.py

```diff
@@ -5,6 +5,7 @@
 from django.urls import reverse
 
 from casexml.apps.phone.models import SyncLogSQL
+from corehq.apps.domain.auth import FORMPLAYER
 from corehq.apps.domain.shortcuts import create_domain
 from corehq.apps.domain.tests.test_utils import delete_all_domains
 from corehq.apps.users.models import CommCareUser, WebUser
```

Comment:
I forgot to remove `WebUser` (this line) and `uuid` (line 1) when I made my change, leaving lint errors. Any chance you could tack that cleanup on this PR? Or I can do it in a separate PR if that would be more handy.